.github/workflows/update_gloo_apis_pr.yaml

```diff
@@ -1,11 +1,9 @@
 on:
+  workflow_dispatch:
   push:
     branches:
     - "update-gloo-apis"
-  pull-request:
-    branches:
-    - "update-gloo-apis"
-
+    - "gloo-v**"
```

Comment:
This line is for the following automated workflow:

Gloo release v1.7.0-betax
-> Creates a branch in solo-apis called gloo-v1.7.0-betax
-> That branch is part of a PR
-> Someone from Solo has to approve and hit the "run workflow" button
-> Once the PR is merged into master, another github workflow runs that takes the branch name (gloo-v1.7.0-betax) and tags the latest master commit with that name.